bundle/src/main/java/com/adobe/acs/commons/models/injectors/impl/ChildResourceFromRequestInjector.java

```diff
@@ -52,7 +52,7 @@
 @Component(
         service = {Injector.class},
         property = {
-                Constants.SERVICE_RANKING + "=3000"
+                Constants.SERVICE_RANKING + ":Integer=3000"
```

Comment:
I would recommend switching to https://docs.osgi.org/javadoc/osgi.cmpn/7.0.0/org/osgi/service/component/propertytypes/ServiceRanking.html but that requires an update of the `maven-bundle-plugin` to a version >= 4.0

Reply:
comments in pom.xml says that there some issues with 4.x.x version